bot/event_handler.py

```diff
@@ -65,7 +69,10 @@ def _handle_message(self, event):
 
         # Remove mention of the bot so that the rest of the code doesn't need to
         msg_txt = self.clients.remove_mention(msg_txt).strip()
-        # Possible error if hal is mentioned in message to hal?
+
+        # Ensure that we don't go to wit with messages posted by slackbot
+        if event['user'] == "USLACKBOT":
```

Comment:
Would be good to have a set of ignored users and add this user ID to that set. I assume there will be other bots we'll want to ignore in the future.